selfservice/strategy/link/strategy_recovery.go

```diff
@@ -333,6 +333,10 @@ func (s *Strategy) recoveryIssueSession(w http.ResponseWriter, r *http.Request,
 		return s.handleRecoveryError(w, r, f, nil, err)
 	}
 
+	if err := s.d.RecoveryExecutor().PostRecoveryHook(w, r, f, sess); err != nil {
+		return s.handleRecoveryError(w, r, f, nil, err)
```

Comment:
@aeneasr: Is this the proper place for a web-hook, or should it be moved?

Reply:
Yeah that LGTM